.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    env:
+      CC: gcc-10
+      CXX: g++-10
+    steps:
+      - uses: actions/checkout@v2
+      - run: cmake .
+      - run: cmake --build .
+
+  macos:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: cmake .
+      - run: cmake --build .
+
+  android:
```

Comment:
Is there a way to run this before merging? Elsewhere I've seen the [NDK manually installed](https://github.com/AOMediaCodec/libavif/blob/main/.github/workflows/ci-android-jni.yml#L12), is there one available by default?

Reply:
Yes, you can either [activate GitHub Actions here](https://github.com/webmproject/sjpeg/actions) or see [this example run](https://github.com/EwoutH/sjpeg/actions/runs/3341346795) on my fork.

As for the Android NDK, it's using the default one installed on the GitHub Actions [runner-images](https://github.com/actions/runner-images). Currently that's the [NDK 25.1.8937393](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#environment-variables-3).

Reply:
Thanks for the pointers. @vigneshvg maybe the extra NDK install step isn't necessary in libavif.